app/index.js

```diff
@@ -18,7 +18,7 @@ const root = angular
         componentsModule,
         pagesModule
     ])
-    .constant("appConfig", "https://project-5613440220430148247.firebaseio.com/api/v1/")
+    .constant("appConfig", "https://woven-patrol-127817.firebaseio.com/api/v1/")
```

Comment:
`appConfig` is not clear - maybe better something like `BASE_URL`? Also I think that constants naming should be UPPERCASE, what do you think ?